### PR TITLE
add demos for other card types

### PR DESCRIPTION
**Pull request by bjfletcher**
/cc @onishiweb @andygnewman @debugwand @leggsimon @ironsidevsquincy 

I was nervous making some change to the title for card and so I figured I'd spend a quick 30 minutes completing the demos page with the missing card types:

- live blog cards
- concept cards
- video cards
- collection cards

demos page now looks like...

image

### CODE REVIEW

File: demos/main.js [hunk 1]
```diff
@@ -37,6 +38,73 @@ const articleCardPropsBase = {
...
+	flags: {}
+};
+ReactDOM.render(<cards.CollectionCard {...collectionCardProps} />, document.getElementById('collection-card'));
+
+// VIDEO CARD
+
+const videoCardProps = {
+	title: 'GSK to relax intellectual property rules to help developing world',
+	id: '903cc52e-f729-11e5-803c-d27c7117d135',
+	flags: {},
```
Comment (ironsidevsquincy) [inline demos/main.js]: interesting. do the cards break if a `flags` object isn't passed? Might be worth making that optional

**Comment by ironsidevsquincy** [comment]
👍  Excellent!!!

File: demos/main.js (hunk 1 above)
Comment (bjfletcher) [inline demos/main.js]: I'll do that in a separate PR.

**Comment by leggsimon** [comment]
Awesome. It does look like there's an issue with the rightmost Branded Card. With the encoding in the brand?

**Comment by bjfletcher** [comment]
@leggsimon https://github.com/Financial-Times/n-card/pull/174 fixes the encoding issue

**Comment by andygnewman** [comment]
Great idea - good stuff! 👍

**Pull request merged**